src/App.tsx

```diff
@@ -1,26 +1,145 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoItem } from './components/TodoItem';
+import { FilterType } from './types/FilterType';
+import { ErrorTypes } from './types/ErrorTypes';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorTypes | null>(null);
```

Comment:
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorTypes>(ErrorTypes.Default);
```

Create `Default` option in your `enum`